### PR TITLE
Add io.photoflare.photoflare

### PR DESCRIPTION
**Pull request by hfiguiere**
See PhotoFlare/photoflare#238

See https://discourse.flathub.org/t/photoflare-flatpak-request/125

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build io.photoflare.PhotoFlare

**Comment by hfiguiere** [comment]
Patch to appdata submitted upstream: PhotoFlare/photoflare#251

**Comment by hfiguiere** [comment]
No idea why this failed.

**Comment by bilelmoussaoui** [comment]
You're using `io.photoflare.PhotoFlare` as an app-id and the metainfo is installed under `io.photoflare.photoflare.appdata.xml`. One of the two needs to be updated

**Comment by hfiguiere** [comment]
flatpak-builder once again said nothing.

**Comment by hfiguiere** [comment]
bot, build io.photoflare.photoflare

**Comment by hfiguiere** [comment]
forget that last attempt.

**Comment by hfiguiere** [comment]
bot, build io.photoflare.photoflare

File: io.photoflare.photoflare.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "app-id": "io.photoflare.photoflare",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.13",
```
Comment (bilelmoussaoui) [inline io.photoflare.photoflare.json]: Not a blocker, but have you tried building it using 5.14?

File: io.photoflare.photoflare.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  "modules": [
+    {
+      "name": "graphicsmagick",
+      "builddir": true,
+      "config-opts": [
+        "--enable-shared"
+      ],
+      "cleanup": [
+	"/bin/GraphicsMagick*config"
```
Comment (bilelmoussaoui) [inline io.photoflare.photoflare.json]: Nitpick: indentation

**Comment by bilelmoussaoui** [comment]
Is upstream interested in (co-)maintaining this? Otherwise, looks good! thanks

File: io.photoflare.photoflare.json (hunk 2 above)
Comment (hfiguiere) [reply io.photoflare.photoflare.json]: fixing. I forgot to untabify the json.

File: io.photoflare.photoflare.json (hunk 1 above)
Comment (hfiguiere) [reply io.photoflare.photoflare.json]: I actually didn't know it was out. Will do.

**Comment by hfiguiere** [comment]
bot, build io.photoflare.photoflare

**Comment by hfiguiere** [comment]
> Is upstream interested in (co-)maintaining this? Otherwise, looks good! thanks

commented on PhotoFlare/photoflare#238

**Pull request closed without merging**